next.config.js

```diff
@@ -2,6 +2,10 @@
 const withTM = require("next-transpile-modules")(["react-timezone-select"]);
 const { i18n } = require("./next-i18next.config");
 
+const withBundleAnalyzer = require("@next/bundle-analyzer")({
```

Comment:
:wave: @zomars next/bundle-analyzer is in dev dependency but it's required to run the app.
It made the app crash if dev dependencies are not keep for runtime.

And thanks for this work, it will help !